core/python_utils.py

```diff
@@ -393,19 +393,14 @@ def _recursively_convert_to_str(value):
         return value
 
 
-def yaml_from_dict(dictionary, width=80):
-    """Gets the YAML representation of a dict.
-
-    Args:
-        dictionary: dict. Dictionary for conversion into yaml.
-        width: int. Width for the yaml representation, default value
-            is set to be of 80.
-
-    Returns:
-        str. Converted yaml of the passed dictionary.
-    """
-    dictionary = _recursively_convert_to_str(dictionary)
-    return yaml.safe_dump(dictionary, default_flow_style=False, width=width)
+def reraise_exception():
+    """Reraise exception with complete stacktrace."""
+    # TODO(#11547): This method can be replace by 'raise e' after we migrate
+    # to Python 3.
+    # This code is needed in order to reraise the error properly with
+    # the stacktrace. See https://stackoverflow.com/a/18188660/3688189.
+    exec_info = sys.exc_info()
+    six.reraise(exec_info[0], exec_info[1], tb=exec_info[2])
```

Comment:
Why is this added?

Reply:
I had removed the yaml_from_dict from python utils , however this function was present in my branch but not in upstream develop so it is coming out to be as added (I did not add it , previously present) removed it in my branch

Reply:
it was removed in #13992